### PR TITLE
Checking whether SonarQube flags this now

### PR DESCRIPTION
**Pull request by Bert-R**
With the reconfigured PR builder.

### CODE REVIEW

File: core/src/main/java/nu/yona/server/rest/GlobalExceptionMapping.java [hunk 1]
```diff
@@ -47,6 +48,8 @@
 	@ResponseBody
 	public ErrorResponseDto handleOtherException(Exception exception, HttpServletRequest request)
 	{
+		Optional<String> opt = Optional.empty();
+		logger.debug(opt.get());
 		return logUnhandledExceptionAndCreateErrorDto("completed with unknown exception", exception, request);
```
Comment (yonabuild) [inline core/src/main/java/nu/yona/server/rest/GlobalExceptionMapping.java]: MAJOR Call "opt.isPresent()" before accessing the value. [rule](http://build.dev.yona.nu:9000/coding_rules#rule_key=squid%3AS3655)

**Comment by yonabuild** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by Bert-R** [comment]
It works now. Closing this PR.

File: core/src/main/java/nu/yona/server/rest/GlobalExceptionMapping.java (hunk 1 above)
Comment (aristotelos) [reply core/src/main/java/nu/yona/server/rest/GlobalExceptionMapping.java]: 👍

**Pull request closed without merging**